api/src/main/java/org/pmiops/workbench/api/UserAdminController.java

```diff
@@ -144,4 +146,11 @@ public ResponseEntity<ListEgressBypassWindowResponse> listEgressBypassWindows(Lo
         new ListEgressBypassWindowResponse()
             .bypassWindows(userAdminService.listAllEgressBypassWindows(userId)));
   }
+
+  @Override
+  public ResponseEntity<ListUserDisabledEventsResponse> listUserDisabledEvents(Long userId) {
```

Comment:
looks like we need this method to be defined in `UserAdminApiStub` for the UI unit tests to pass